pom.xml

```diff
@@ -17,7 +17,7 @@
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <!-- Maven plugins -->
         <maven-compiler-plugin.version>3.7.0</maven-compiler-plugin.version>
-        <tor-binary.version>b9c6227</tor-binary.version>
+        <tor-binary.version>6d1fd95</tor-binary.version>
```

Comment:
FYI: This is the commit ID of the last tor-binary commit: https://github.com/bisq-network/tor-binary